src/sass/_common/footer.scss

```diff
@@ -108,8 +107,8 @@
         }
     }
     .trustlogo2 {
-        float: right;
-        width: 175px;
+        width: 180px;
+        margin-right: -15px;
```

Comment:
please use center alignment class rather than hardcoded pixel of margin

Reply:
it has an internal margin out of our control, know it is not good idea but cannot find other way with this third-party widget to force be right align with next line in footer.